Robin-rboynosky/Homework-1/fizzbuzz/app.js

```diff
@@ -0,0 +1,51 @@
+
+// Relying on your newfound knowledge of loops, combine loops and if/else statements together and incrementally build the common fizzbuzz loop.
+
+// - In the loop, every time a number is divisible by **3**, instead of logging the number itself, the word "fizz" should appear.
+// - If the number is divisible by  **5**, the word "buzz" should be logged.
+// - If the number is divisible by both **3** and  **5**, then the word "fizzbuzz" should be logged.
+
+// Follow the steps below.
+
+// ##### Step 1:
+// Construct a for loop that iterates through, and `console.log()`'s out, numbers 1 - 100.
+
+
+
+// ##### Step 2:
+// Add an `if/else` statement that logs the string `"fizz"` if the value being iterated over is divisible by `3`; otherwise, log out the value.
+
+
+
+// ##### Step 3:
+// Add an `else if` clause that logs the string `"buzz"` if the value being iterated over is divisible by `5`.
+
+
+
+// ##### Step 4:
+// Add an additional `else if` clause that logs the string `"fizzbuzz"` if the value being iterated over is divisible by both `3` and `5`.
+
+
+//Tried this
+/*while (number <= 100) {
+    console.log(number ++)
+    if (number % 3 === 0) {
+        console.log("Fizz")
+    } else (number % 5 === 0) 
+        console.log("Buzz")
+};*/
+
+//for (number = 0; number <= 100; number = number ++);
+  //console.log(number);
+
+  let number =0;
+
+for (number =0; number <= 100; number++) {
+    if (number % 15 === 0)
```

Comment:
I think here you want to be checking that the number is divisible by 3 and also divisible by 5 as opposed to being divisible by 3 times 5.